.github/workflows/external-contributors.yml

```diff
@@ -0,0 +1,43 @@
+name: "CI: Mention external contributors"
+on:
+  pull_request:
+    types:
+      - closed
+    branches:
+      - develop
+
+jobs:
+  external_contributor:
+    name: External Contributors
+    runs-on: ubuntu-20.04
+    if: |
+      github.event.pull_request.author_association != 'COLLABORATOR'
+      && github.event.pull_request.author_association != 'MEMBER'
+      && github.event.pull_request.author_association != 'OWNER'
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          ref: ${{ github.head_ref }}
+      - name: Set up Node
+        uses: actions/setup-node@v4
+        with:
+          node-version-file: 'package.json'
+          cache: 'yarn'
+
+      - name: Install dependencies
+        run: yarn install --frozen-lockfile
+
+      - name: Add external contributor to CHANGELOG.md
+        uses: ./dev-packages/external-contributor-gh-action
+        with:
+          name: ${{ github.event.pull_request.user.login }}
+      - name: Create PR with changes
+        uses: peter-evans/create-pull-request@v6
```

Comment:
I know we didn't have this before but can we pin the action to an exact SHA considering this is an inofficial action?